v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/ErrorConverters.java

```diff
@@ -505,4 +509,92 @@ public PubsubMessage apply(KV<T, Map<String, String>> kv) {
       }
     }
   }
+
+  /**
+   * {@link WriteStringMessageErrors} transform writes strings error messages to file system using
```

Comment:
Can you change the documentation to align to latest class definition?

Reply:
Done. I missed this, thank you.